Explore:
  ros__parameters:
    speed: 0.26 # Float max 0.26
    max_steering_angle: 30.0 # Used to calculate the steering angle around corners
    lookahead_distance: 0.34 # Distance to lookahead in the path for pure pursuit
    prioritize_direction: False # not used
    near_obstacle: 0.27 # Used to detect walls and obstacles
    distance_to_waypoint_threshold: 0.3 # Distance to path waypoint to consider reached
    control_loop_period: 0.05 # How often to execute the navigation commands
    non_progress_threshold: 3 # How many tries until the path will reset
    progress_timeout_seconds: 5.0 # Seconds to trigger checking for navigation towards goal

Computation:
  ros__parameters:
    computation_frequency: 0.1 # Executions for the computations
    offset_to_the_boundary_points: 6 # Offset to inflate the walls keep below 6
    occupied_threshold: 100 # Cells to consider as occupied Integer
    cluster_size: 8 # Define minimum amount of samples to consider a cluster
    eps: 7 # Distance between clusters
    distance_to_cluster_threshold: 0.5 # Update the cluster navigation goal when the robot are this distance to the cluster
    use_furthest_point: True # Set False to select the furthest point in the nearest cluster, True for centroid

Draw:
  ros__parameters:
    publish_period: 0.5 # The period which to publish markers in RVIZ

